src/utils/graphql/fragments/template.fragment.ts

```diff
@@ -8,5 +8,21 @@ export default gql`
     isLinear
     startMessage
     submissionMessage
+    templateCategory {
+      title
+      icon
+    }
```

Comment:
Oh nice! We had one [Epic#81](https://github.com/openmsupply/application-manager-web-app/issues/539) just for categories work - I guess we can mark it as done (move to awaiting public docs actually) after this PR is merged - possibly nice to have some internal docs created regarding that too